core/src/test/java/com/graphhopper/routing/subnetwork/PrepareRoutingSubnetworksTest.java

```diff
@@ -34,32 +39,15 @@
 import java.util.List;
 
 import static com.graphhopper.routing.weighting.TurnCostProvider.NO_TURN_COST_PROVIDER;
-import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
 /**
  * @author Peter Karich
  */
 public class PrepareRoutingSubnetworksTest {
 
```

Comment:
I tried cleaning up this test, but honestly I still do not like it because it does not really systematically test what PrepareRoutingSubnetworks (now) does. I will probably rewrite it from scratch.

Reply:
Ok I improved the test a bit further in my last commit, now it also includes a test with turn restrictions and I think it is in a good state.

Reply:
Nice